llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -2114,21 +2114,19 @@ void VPlanTransforms::addActiveLaneMask(
     HeaderMask->replaceAllUsesWith(LaneMask);
 }
 
-/// Try to convert \p CurRecipe to a corresponding EVL-based recipe. Returns
-/// nullptr if no EVL-based recipe could be created.
+/// Try to optimize a \p CurRecipe masked by \p HeaderMask to a corresponding
+/// EVL-based recipe without the mask. Returns nullptr if no EVL-based recipe
```

Comment:
```suggestion
/// EVL-based recipe. Returns nullptr if no EVL-based recipe
```
The mask still may be generated